### ISSUE TITLE
For readability, users should see connections from a multiple-output activity more intuitively placed/styled

### ISSUE THREAD

**Issue by tomy2105** [elsa-workflows/elsa-core#781]
Activities with more than one output seem to order subsequent activities in first added leftmost fashion.
Which tends to be confusing and hard to follow if you add activity for right output first (lines are crossed and hard to follow).
The more outputs, the more it can become unreadable :(.

In the image below iterate is connected to set var activity and done to http response activity.
![OutputOrder](https://user-images.githubusercontent.com/7658839/112009421-a4830480-8b26-11eb-8d21-9c0de36f4394.gif)

**Comment by craigfowler** [comment]
Agreed, I can see how what you have there is pretty confusing!

**Comment by tomy2105** [comment]
There is a second issue related to this where there is no cross overlap of lines (left output goes to right child activity and right output to left child activity) but lines simple overlap when more than 3 outputs even in case child activites are sorted same as outputs.

![Image 2](https://user-images.githubusercontent.com/7658839/114876851-06215080-9dff-11eb-88e5-c09bfe2b472e.png)

**Comment by sfmskywalker** [comment]
Fixed:

![image](https://user-images.githubusercontent.com/938393/116413684-24903e80-a838-11eb-9ded-3bd998353ea4.png)

![image](https://user-images.githubusercontent.com/938393/116414063-746f0580-a838-11eb-9cc2-51380613daa4.png)
